### ISSUE TITLE
Nan or Inf found in input tensor.

### ISSUE THREAD

**Issue by Amazingren** [swathikirans/GSM#1]
Hellow, I just want to train your methods in my machine with something-v1 dataset,
And in my machine:
(1)Pytorch-1.2
(2)Python-3.7
(4)TensorboardX-also is suitable
(5)4-Gpus

However, when I run the train scripts, Loss is always 'nan' and the outputing the warning:
`Nan or Inf found in input tensor.`
I have tried to handle this nut by:
(1)turn the learning rate smaller, but this not work;
(2)check the 'loss.backward()',before and after the backpropragate , I print the losss , I find that before backpropragate, the loss is normal, while after this operation, loss=nan

What's more ,
(3)I also checked the 'datasets_video.py', It appears that in your fuction 'return_something():' it doesn't need the file  'filename_categories.txt' compared with other methods, I am also confused about this. 

So could you please show some light to me about those two questions.
Thx very much!

**Comment by swathikirans** [comment]
Hi,
I am not sure why this error is occurring. Does the loss turn to NaN after the first parameter update or after few iterations?
Did you check the input before it is applied to the model? You can try to run the model without updating the parameters and see if the loss is becoming NaN. Then, it could be a problem with the input.

For the 'filename_categories.txt' file, it is not needed anywhere in code.

**Comment by swathikirans** [comment]
Closing due to inactivity.
